db/migrate/20170813163239_add_columns_to_events.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsToEvents < ActiveRecord::Migration[5.0]
+  def change
+  	add_column :events, :source_id, :string, index: true, unique: true 
+  	add_column :events, :source, :string, index: true
```

Comment:
The rails convention for polymorphic associations is for this column to end in `_type`, i.e. `source_type`.  `source_id` and `source_type` go together.

Technically, since we are not applying this is as a polymorphic association (because the `source_id` is a `string` instead of an `integer`), you don't **have to** change it.  But it would be nice :)  If you do, be sure to find everywhere it is used and replace.

Reply:
How big is the events table? Will there be any downtime associated with this PR? @rickr (You might not be the best person to tag for this question)

Reply:
@mwagz - Good question!  We are good with this merging at any time, no downtime required.  

The `event` table is relatively small.  And when this `rake` task is running daily, @leenyburger has checked and it will be pulling a small number of events in per day.    

@leenyburger , can you ballpark that number for us.

Reply:
Yup, 20-30 events per day